xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd

```diff
@@ -500,13 +500,26 @@ Rail transport, Roads and Road transport
 		<xsd:restriction base="xsd:string">
 			<xsd:enumeration value="scooter"/>
 			<xsd:enumeration value="eScooter"/>
-			<xsd:enumeration value="cycle"/>
+			<xsd:enumeration value="cycle">
+				<xsd:annotation>
+					<xsd:documentation>All bicycles.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
 			<xsd:enumeration value="pedalCycle">
 				<xsd:annotation>
-					<xsd:documentation/>
+					<xsd:documentation>Regular bicycle without motor or pedal support.</xsd:documentation>
```

Comment:
I still don't agree. I read the current annotation as: "pedalCycle = bicycle without pedal support or bicycle without motor".

Reply:
@Aurige  I am also unhappy with the definition, but that' how it was written. In my view a pedalCycle has pedals (a "normal" bicycle).

Reply:
Better: Regular bicycle with pedals (may have motor support)?

Reply:
I think "Non-motorised cycle." would be a correct text. 

`eCycle` is a (electrically) motorised bicycle (having pedals and a motor), `pedalCycle` a non-motorised bicycle, `cycle` includes both.

Reply:
I was understanding the "without pedal support" as "without any help for pedalling"
Maybe: _Non-motorised (without any kind of pedalling aid) cycle_